### PR TITLE
Allow disabling of curator

### PR DESCRIPTION
**Pull request by jjungnickel**
This may be useful when your cluster does not have the batch APIs enabled yet.

### CODE REVIEW

File: templates/es-curator-config.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{{ if .Values.curator.enabled }}
```
Comment (sagikazarmark) [inline templates/es-curator-config.yaml]: You could suppress newlines by using {{- <something> -}} tags.

File: values.yaml [hunk 2]
```diff
@@ -62,6 +62,7 @@ master:
     NUMBER_OF_MASTERS: "2"
 
 curator:
+  enable: false
```
Comment (sagikazarmark) [inline values.yaml]: This technically breaks previous installations, so a note somewhere might make sense about that.

Comment (jjungnickel) [reply values.yaml]: Yeah, I think it's better to leave it at the previous implicit default of `true`

File: templates/es-curator-config.yaml (hunk 1 above)
Comment (jjungnickel) [reply templates/es-curator-config.yaml]: Of course, thanks

**Comment by clockworksoul** [comment]
Sorry it's taken so long for me to take a look. LGTM!

**Comment by clockworksoul** [comment]
Sorry it's taken so long for me to take a look. LGTM!

**Pull request merged**